data_structures/src/error.rs

```diff
@@ -278,6 +278,14 @@ pub enum TransactionError {
         max_weight: u32,
         dr_output: Box<DataRequestOutput>,
     },
+    #[fail(
+        display = "The reward-to-collateral ratio for this data request is {}, but must be greater than {}",
+        reward_collateral_ratio, required_reward_collateral_ratio
+    )]
```

Comment:
As the value printed here is the ratio, and not the percentage, then the threshold works the other way:
```suggestion
    #[fail(
        display = "The reward-to-collateral ratio for this data request is {}, but must be equal or less than {}",
        reward_collateral_ratio, required_reward_collateral_ratio
    )]
```